### DISCUSSION TITLE
Supporting WinUI3 in HoloLens 2

### DISCUSSION THREAD

**Discussion by laultman** [StereoKit/StereoKit#699] [General]
Closed

**Comment by maluoi** [comment]
> First question, why? Because SK is shifting and using C# greater than v7.3 which is the last supported version in UWP, and I don't want to fork SK. This means that SK will not support HoloLens 2 UWP.

Just to be clear, StereoKit does, and will continue to support UWP. SK uses .NET Standard 2.0 specifically to be UWP compatible, and currently ships ARM, ARM64, and x64 binaries specifically for UWP. [As I mentioned before](https://github.com/StereoKit/StereoKit/discussions/682#discussioncomment-6531958), I will continue to support UWP as long as it remains relevant.

How are you intending to use WinUI 3 and StereoKit together? And why would you choose to use WinUI 3 when it [explicitly does not support HoloLens](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/migrate-to-windows-app-sdk/what-is-supported)?

It might also be helpful for you to list what you need more clearly. It's difficult for me to pull action items from that much text.

**Reply by maluoi** [reply]
One thought that did come to my mind was that you might be interested in playing around with SK's default asset copy behavior. Especially since you're seeing duplicates! The [relevant variables are over here](https://github.com/StereoKit/StereoKit/blob/develop/StereoKit/StereoKit.props#L4-L6).
